Dockerfile

```diff
@@ -25,9 +20,14 @@ RUN make dm-worker dm-master dmctl
 
 FROM alpine:3.10
 
+# keep compatibility
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-worker /dm-worker
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dm-master /dm-master
 COPY --from=builder /go/src/github.com/pingcap/dm/bin/dmctl /dmctl
-COPY --from=builder /opt/tools/bin/mydumper /mydumper
```

Comment:
`make check` may need this binary, please check if CI could work

Reply:
now dm not need this bin, docker image already build here https://github.com/pingcap/dm/pull/1902/checks?check_run_id=3129978217